pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/structure/UniqueField.java

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.dsm.db.structure;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface UniqueField {
+    boolean value();
```

Comment:
The value does not seem to provide anything useful. Can you create an annotation without it?